acsg.js

```diff
@@ -62,53 +62,53 @@ var acsg = {}  // Module namespace
 acsg.Browser = (function () {
 
   var Browser = function (game, opts) {
-      if (!(this instanceof Browser)) {
-          return new Browser(game, opts)
-      }
-      this.game = game
-      // Seed background animation RNG.
-      var backgroundRngFunc = seedrandom(this.now())
-      this.rBackground = new Rands(backgroundRngFunc)
-      this.scoreboard = document.getElementById('score')
-      this.bonus = document.getElementById('dollars')
-      this.clock = document.getElementById('clock')
-      this.data = []
-      this.background = []
-      this.opts = opts
-
-      for (var i = 0; i < opts.ROWS; i++) {
-        for (var j = 0; j < opts.COLUMNS; j++) {
-          this.data.push(BLACK)
-          this.background.push(BLACK)
-        }
+    if (!(this instanceof Browser)) {
+      return new Browser(game, opts)
+    }
+    this.game = game
+    // Seed background animation RNG.
+    var backgroundRngFunc = seedrandom(this.now())
+    this.rBackground = new Rands(backgroundRngFunc)
+    this.scoreboard = document.getElementById('score')
+    this.bonus = document.getElementById('dollars')
+    this.clock = document.getElementById('clock')
+    this.data = []
+    this.background = []
+    this.opts = opts
+
+    for (var i = 0; i < opts.ROWS; i++) {
+      for (var j = 0; j < opts.COLUMNS; j++) {
+        this.data.push(BLACK)
+        this.background.push(BLACK)
       }
+    }
 
-      this.pixels = grid(this.data, {
-        root: document.body,
-        rows: opts.ROWS,
-        columns: opts.COLUMNS,
-        size: opts.BLOCK_SIZE,
-        padding: opts.BLOCK_PADDING,
-        background: GRAY,
-        formatted: true
-      })
+    this.pixels = grid(this.data, {
+      root: document.body,
+      rows: opts.ROWS,
+      columns: opts.COLUMNS,
+      size: opts.BLOCK_SIZE,
+      padding: opts.BLOCK_PADDING,
+      background: GRAY,
+      formatted: true
+    })
 
-      if (this.opts.INCLUDE_HUMAN) {
-        this._bindKeys()
-      }
+    if (this.opts.INCLUDE_HUMAN) {
+      this._bindKeys()
+    }
   }
 
   Browser.prototype.now = function () {
     return performance.now()
   }
 
   Browser.prototype.updateScoreboard = function (ego) {
-    this.scoreboard.innerHTML = ego.score
-    this.bonus.innerHTML = ego.payoff.toFixed(2)
+    if (this.scoreboard !== null) {this.scoreboard.innerHTML = ego.score}
```

Comment:
In what context were these elements null?

Reply:
This isn't really a linter fix, but a real world fix. The HTML for the scoreboard and friends is set in the Dallinger experiment template and is therefore not guaranteed to be in place.